schema/artist/index.js

```diff
@@ -193,6 +195,21 @@ export const ArtistType = new GraphQLObjectType({
           )
         },
       },
+      auctionResults: {
```

Comment:
We went with 🐫 on this- @alloy any special reason why we use snake case for schema/fields?

Reply:
Ooohhhhh!1!! I absolutely :heart: going 🐫!

I can only assume that the historical reason for snake case is that early schema design was all 1:1 mapping API responses.

In the context of GraphQL and JS clients, it’s more idiomatic to use :camel:, but seeing as we wanted people using different languages (e.g. Ruby) to also start using GraphQL, I felt it might have been too subjective a topic to broach.

[I mentioned](https://artsy.slack.com/archives/C1HH3KNJG/p1511269554000220) that for the next front-end meeting I wanted to discuss this moment in time (around e.g. schema stitching) to see if we can maybe reboot the schema somewhat, we can make this part of that discussion 👍

Reply:
I don't think we should be optimizing for JS clients, but I do think we should adopt JSON conventions. While I'm familiar with JS's conventions, my 30 seconds of research didn't reveal whether that's actually adopted for JSON. Anyone know better?

Reply:
Good question! Several comments I’ve found around the web linked to [the Google JSON Style Guide](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format), which says to follow the JS convention. (_Probably_ because JSON is **JavaScript** Object Notation?)